src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -101,6 +102,55 @@
      */
     private BlockPos currentPatrolPoint = null;
 
+    /**
+     * The value of the speed which the guard will move.
+     */
+    private static final double ATTACK_SPEED = 0.8;
+
+    /**
+     * Indicates if strafing should be moving backwards or not.
+     */
+    private boolean strafingBackwards = false;
+
+    /**
+     * Indicates if strafing should be clockwise or not.
+     */
+    private boolean strafingClockwise = false;
+
+    /**
+     * Amount of time strafing is able to run.
+     */
+    private int strafingTime = -1;
+ 
+    /**
+     * Amount of time the guard has been in one spot.
+     */
+    private int timeAtSameSpot = 0;
+
+    /**
+     * Amount of time left until guard can attack again.
+     */
+    private int attackTime = 0;
+
+    /**
+     * Number of ticks the guard has been way to close to target.
+     */
+    private int toCloseNumTicks = 0;
```

Comment:
### Errors:
- Field initialization to 0 is redundant 

### Warnings:
- Instance field name toCloseNumTicks doesn't match regex 'm_[a-z][A-Za-z\d]*'